tempesta_fw/sock_srv.c

```diff
@@ -680,12 +705,12 @@ tfw_cfgop_sticky_sess(TfwCfgSpec *cs, TfwCfgEntry *ce, unsigned int *use_sticky)
 		TFW_ERR_NL("Invalid number of arguments: %zu\n", ce->val_n);
 		return -EINVAL;
 	}
-	if (!ce->val_n) {
+	if (tfw_cfg_is_dflt_value(ce)) {
+		*use_sticky = 0;
+	} else if (!ce->val_n) {
 		*use_sticky = TFW_SRV_STICKY;
 	} else if (!strcasecmp(ce->vals[0], "allow_failover")) {
 		*use_sticky = TFW_SRV_STICKY | TFW_SRV_STICKY_FAILOVER;
-	} else if (!strcasecmp(ce->vals[0], TFW_CFG_DFLT_VAL)) {
```

Comment:
As i see, `TFW_CFG_DFLT_VAL` is not used anywhere now, and can be replaces with `NULL`.